lib/cli/src/Cardano/CLI.hs

```diff
@@ -765,6 +767,16 @@ databaseOption = optionT $ mempty
     <> metavar "DIR"
     <> help "use this directory for storing wallets. Run in-memory otherwise."
 
+-- | [--listen-address=HOSTSPEC], default: 127.0.0.1
+hostPreferenceOption :: Parser HostPreference
+hostPreferenceOption = option str $ mempty
+    <> long "listen-address"
```

Comment:
what about calling this just `host` ?

Reply:
You can rename it if you like, but "listen address" is a common and well understood term for network server applications. "host" is too vague in my opinion.